Add transmit pulse model used for match filtering and then broadband TS and Sv calculation (#65)
ices-publications/SONAR-netCDF4: docs/crr341.adoc

=== Type 4

Type 4 conversion equations are intended for data produced by Simrad EK80 and ES80 broadband echosounders when data is recorded with complex samples.

EK80 and ES80 can record data using EK60 Power and Angle format and Type 3 equations can then be used.

In order to take advantage of the broadband mode of the EK80 echosounder, the backscatter complex amplitude samples are stored in the _backscatter_r_ and the _backscatter_i_ variable with data type _float_ for each sub-beam corresponding to each sub-array of the transducer (<<Split-aperture beams>>).

In order to increase signal-to-noise ratio and resolution along the beam axis, pulse compression is used for each sub-beam by convolving the complex backscatter received signal with a complex conjugated and time reversed version of the theoretical transmit and filtered pulse  (_transmit_pulse_model_)

Noting latexmath:[n], the sample index in the discrete time domain, the pulse compressed complex samples for each sub-beam denoted latexmath:[y_{pc}] are then equal to:

[stem]
++++
\begin{equation}
y_{pc}(n) = \frac{y_{rx}(n)*y_{tx,th}^*(-n)}{||y_{tx,th}||^2_2},
\end{equation}
++++

> where latexmath:[y_{rx}] is the complex received signal stored in he _backscatter_r_ and the _backscatter_i_ variables and latexmath:[y_{tx,th}] is the complex theoretical transmit and filtered signal stored in _transmit_pulse_model_r_ and _transmit_pulse_model_i_ variables

Reviewer: ```suggestion
where latexmath:[y_{rx}] is the complex received signal stored in the _backscatter_r_ and the _backscatter_i_ variables and latexmath:[y_{tx,th}] is the complex theoretical transmit and filtered signal stored in _transmit_pulse_model_r_ and _transmit_pulse_model_i_ variables
```

Author: Agreed and corrected

The transceiver measures voltage over a load, latexmath:[z_{\textrm{rx,e}}], connected in series with the transducer impedance, latexmath:[z_{\textrm{td,e}}]. The number of sub-beams latexmath:[N_{u}] are stored as the _subbeam_ dimension in the backscatter variables in <<sonarBeamGroupTable>> and the matched filter received electric power is then equal to :

[stem]
++++
\begin{equation}
p_{\textrm{rx,e}}(n)= \left( \frac{| \sum_{u = 1}^{N_{u}} y_{pc}(n,u)|}{2\sqrt{2}} \right)^2 \left( \frac{|z_{\textrm{rx,e}}+z_{\textrm{td,e}}|}{z_{\textrm{rx,e}}} \right)^2 \frac{1}{|z_{\textrm{td,e}}|}.
\end{equation}
++++

> match filtered recieved electric power is transformed into calibrated target strength using:

Reviewer: ```suggestion
The match filtered received electric power is transformed into calibrated target strength using:
```

Author: Agreed and corrected

++++

\begin{equation}

> TS(f) = 10\log_{10}(p_{\textrm{rx,e}}(f)) + 40\log_{10}(r) + 2\alpha(f) r

Reviewer: This is the broadband equation - should also include the equation for narrowband complex data from the EK80.

- 10\log_{10}\left( \frac{P_t\lambda^{2}}{16\pi^2} \right)- 2G_0(f),
\end{equation}

++++

> where p_{\textrm{rx,e}}(f) is the Fourier transform of the match filtered recieved electric power at frequency f, latexmath:[r] is the range from the transducer to the target [m], latexmath:[\alpha] the acoustic absorption at the frequency f [dB m^-1^], latexmath:[P_t] the transmitted power setting (_transmit_power_ variable), latexmath:[\lambda] the acoustic wavelength [m], and latexmath:[G_0] the on-axis gain (variable _transducer_gain_) at the frequency f from the calibration exercise. Beam pattern compensation for echoes that do not arrive on the transducer axis is done in addition to this equation (if such angles can be estimated, as can be done with split-aperture transducers). Volume backscatter strength is obtained from a similar equation:

Reviewer: ```suggestion
where latexmath:[p_{\textrm{rx,e}}(f)] is the Fourier transform of the match filtered recieved electric power at frequency f, latexmath:[r] is the range from the transducer to the target [m], latexmath:[\alpha] the acoustic absorption at the frequency f [dB m^-1^], latexmath:[P_t] the transmitted power setting (_transmit_power_ variable), latexmath:[\lambda] the acoustic wavelength [m], and latexmath:[G_0] the on-axis gain (variable _transducer_gain_) at the frequency f from the calibration exercise. Beam pattern compensation for echoes that do not arrive on the transducer axis is done in addition to this equation (if such angles can be estimated, as can be done with split-aperture transducers). Volume backscatter strength is obtained from a similar equation:
```

Author: Agreed and corrected

[stem]
++++
\begin{equation}
S_v(f))= 10\log_{10}(p_{\textrm{rx,e}}(f)) + 20\log_{10}(r) + 2\alpha(f) r - 10\log_{10}\left(\frac{P_t \lambda^2 c \psi (\frac{f_n}{f})^2 \tau_e}{32 \pi^2} \right) - 2G_0(f),
\end{equation}
++++

where latexmath:[c] is sound speed [m s^-1^], latexmath:[\psi] the equivalent beam angle at the nominal frequency (variable _equivalent_beam_angle_), and latexmath:[\tau_e] the effective pulse duration (variable _receive_duration_effective_).

> The effective pulse duration is computed for EK80 and ES80 from ....

Reviewer: I assume you will add more here, including how to calculate the theoretical transmit pulse?

Author: Agreed and completed
